### ISSUE TITLE
All article related object should be provided by it is controller and api

### ISSUE THREAD

**Issue by vahid-forghani** [amezenin/encyclopedia-frontend#4]
https://github.com/amezenin/encyclopedia-frontend/blob/be6437b91bfedc1cc2d72ed96b985311747e91e9/src/app/article/article.service.ts#L29

Here you are returning the articles list by users api which is not good

**Comment by amezenin** [comment]
Yes, it is mistake in url mapping in article-controller. Those methods are in the right controller. I will fix the links in back-end.
